src/Polytopes/Polyhedron/standard_constructions.jl

```diff
@@ -681,3 +681,30 @@ Essentially, one can read the
 upperbounds.
 """
 upper_bound_theorem(d::Int,n::Int) = Polyhedron(Polymake.polytope.upper_bound_theorem(d,n))
+
+
+@doc Markdown.doc"""
+    polarize(P::Polyhedron)
+
+Return the polar dual of the polyhedron `P`, consisting of all linear functions
+whose absolute value on `P` does not exceed 1.
```

Comment:
```suggestion
whose evaluation on `P` does not exceed 1.
```
there should be no absolute value here